### PR TITLE
Feature/fe 45 view UI improve settings page

### PR DESCRIPTION
**Pull request by emmanueletukudo**
### Scope & Purpose

This PR adds tooltips with helpful information to the options on the View UI settings page with a little UI enhancement.

- [ ] :hankey: Bugfix
- [x] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*
  - [ ] Backport for 3.7: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [x] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx [hunk 1]
```diff
@@ -1,22 +1,28 @@
 /* global frontendConfig */
 
-import { cloneDeep } from 'lodash';
-import React, { useEffect, useReducer, useRef, useState } from 'react';
-import Textbox from '../../components/pure-css/form/Textbox';
+import { cloneDeep } from "lodash";
```
Comment (adityamukho) [inline js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx:3]: Formatting changes

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
+  getNumericFieldValue,
+  getReducer,
+  isAdminUser as userIsAdmin,
   usePermissions
-} from '../../utils/helpers';
-import { DeleteButton, SaveButton } from './Actions';
-import { buildSubNav, postProcessor, useView } from './helpers';
+} from "../../utils/helpers";
+import { DeleteButton, SaveButton } from "./Actions";
+import { buildSubNav, postProcessor, useView } from "./helpers";
```
Comment (adityamukho) [inline js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx:15]: More formatting changes.

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx [hunk 3]
```diff
@@ -25,129 +31,203 @@ const ViewSettingsReactView = ({ name }) => {
...
+  return (
+    <div className={"centralContent"} id={"content"}>
+      <div
+        id={"modal-dialog"}
+        className={"createModalDialog"}
+        tabIndex={-1}
+        role={"dialog"}
+        aria-labelledby={"myModalLabel"}
+        aria-hidden={"true"}
+      >
```
Comment (adityamukho) [inline js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx:72]: Everything up to here is just reformats.

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx [hunk 4]
```diff
@@ -25,129 +31,203 @@ const ViewSettingsReactView = ({ name }) => {
...
+                        required={true}
+                        disabled={frontendConfig.isCluster || !isAdminUser}
+                      />
+                    </th>
+                    <th className="collectionTh">
+                      <ToolTip
+                        title="The View name (string, immutable)."
+                        setArrow={true}
+                      >
+                        <span className="arangoicon icon_arangodb_info"></span>
```
Comment (adityamukho) [inline js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx:94]: Here's a mix of reformats + new code. Impossible to review like this.

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx [hunk 5]
```diff
@@ -25,129 +31,203 @@ const ViewSettingsReactView = ({ name }) => {
...
+                        type={"number"}
+                        disabled={!isAdminUser}
+                        value={getNumericFieldValue(
+                          formState.cleanupIntervalStep
+                        )}
+                        onChange={getNumericFieldSetter(
+                          "cleanupIntervalStep",
+                          dispatch
+                        )}
+                      />
```
Comment (adityamukho) [inline js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx:115]: Again.

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx [hunk 6]
```diff
@@ -25,129 +31,203 @@ const ViewSettingsReactView = ({ name }) => {
...
+                    </th>
+                    <th className="collectionTh">
+                      <ToolTip
+                        title="The steps to wait before removing unused segments after release of internal resource."
+                        setArrow={true}
+                      >
+                        <span className="arangoicon icon_arangodb_info"></span>
+                      </ToolTip>
```
Comment (adityamukho) [inline js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx:123]: A bit of new code that was almost missed.

**Review by adityamukho — CHANGES_REQUESTED** [review]
Please remove all reformats. A functional PR should not contain reformats, otherwise it is impossible to review.

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx (hunk 4 above)
Comment (emmanueletukudo) [reply js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx:94]: I think VSCode sold me out here again despite disabling Prettier.

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx (hunk 6 above)
Comment (emmanueletukudo) [reply js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx]: Done.

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx (hunk 4 above)
Comment (emmanueletukudo) [reply js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx]: Done.

**Review by adityamukho — APPROVED** [review]
LGTM!

**Comment by adityamukho** [comment]
@KVS85 can you please merge this PR?

**Review by ajurdak — APPROVED** [review]
LGTM.

**Pull request merged**